src/github.com/mozilla-services/pushgo/simplepush/handlers_endpoint.go

```diff
@@ -326,11 +320,13 @@ func (h *EndpointHandler) UpdateHandler(resp http.ResponseWriter, req *http.Requ
 }
 
 // deliver routes an incoming update to the appropriate server.
-func (h *EndpointHandler) deliver(cn http.CloseNotifier, uaid, chid string, version int64, requestID string, data string) (ok bool) {
-	client, clientConnected := h.app.GetClient(uaid)
+func (h *EndpointHandler) deliver(cn http.CloseNotifier, uaid, chid string,
+	version int64, requestID string, data string) (ok bool) {
+
+	w, workerConnected := h.app.GetWorker(uaid)
```

Comment:
How bout w -> worker? I generally don't consider one-letter vars bad if its used in a concise loop, or a small (< 10 lines) function, but the next reference for this is 23 lines down.